src/components/cards/PlatformCard.tsx

```diff
@@ -19,9 +22,16 @@ const PlatformCard = ({
         border="solid 1px "
         borderColor="divider"
         rounded="lg"
+        _hover={{
+          bg: 'divider',
+          color: 'dimmedText',
+        }}
       >
-        <Icon as={icon} boxSize="14" _light={{ color: 'black' }} />
+        <NextLink href={route} passHref>
+          <Icon as={icon} boxSize="14" _light={{ color: 'black' }} />
```

Comment:
Only use `passhref` when the child is an `a` tag
```suggestion
        <NextLink href={route}>
          <Icon as={icon} boxSize="14" _light={{ color: 'black' }} />